general/clients/web-config.md

```diff
@@ -0,0 +1,66 @@
+---
+uid: web-config
+title: Jellyfin Web configuration
+---
+
+The Jellyfin Web default interface can be configured using the `config.json` file in the webroot. Where this is and how to edit it depend on the installation method.
+
+# Editing the configuration
+
+## Debian/Ubuntu/Fedora/CentOS Packages
+
+The configuration can be found at `/usr/share/jellyfin/web/config.json`. This file is registered as a configuration file by the Debian packages, and any changes to the defaults will be handled by `apt` on upgrade.
+
+## Docker
+
+Overriding the default `config.json` can be done with an additional volume parameter to your `docker run` command, e.g.
+
+```
+--volume /path/to/config/web-config.json:/jellyfin/jellyfin-web/config.json
+```
+
+We would recommend obtaining the [latest copy of the file](https://github.com/jellyfin/jellyfin-web/blob/master/src/config.json) to pre-populate your configuration directory before starting Jellyfin for the first time; unlike most other components of this directory, it will not be created automatically on first run.
```

Comment:
Is there any way to preload it? What happens when the config is updated for unstable but people grab it for stable?

Reply:
It should be fine regardless.